org.eclipse.dawnsci.nexus/src/org/eclipse/dawnsci/nexus/NexusConstants.java

```diff
@@ -0,0 +1,147 @@
+/*-
+ * Copyright 2018 Diamond Light Source Ltd.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.dawnsci.nexus;
+
+/**
+ * Constants used to deal with NeXus files
+ */
+public class NexusConstants {
+	/**
+	 * Group attribute to indicate NeXus class
+	 */
+	public static final String CLASS = "NX_class";
+
+	/**
+	 * Dataset attribute name for units
+	 */
+	public static final String UNITS = "units";
+
+	/**
+	 * base classes
+	 */
+	public static final String APERTURE = "NXaperture";
+	public static final String ATTENUATOR = "NXattenuator";
+	public static final String BEAM = "NXbeam";
+	public static final String BEAMSTOP = "NXbeam_stop";
+	public static final String BENDINGMAGNET = "NXbending_magnet";
+	public static final String CAPILLARY = "NXcapillary";
+	public static final String CITE = "NXcite";
+	public static final String COLLECTION = "NXcollection";
+	public static final String COLLIMATOR = "NXcollimator";
+	public static final String CRYSTAL = "NXcrystal";
+	public static final String DATA = "NXdata";
+	public static final String DETECTORGROUP = "NXdetector_group";
+	public static final String DETECTORMODULE = "NXdetector_module";
+	public static final String DETECTOR = "NXdetector";
+	public static final String DISKCHOPPER = "NXdisk_chopper";
+	public static final String ENTRY = "NXentry";
+	public static final String ENVIRONMENT = "NXenvironment";
+	public static final String EVENTDATA = "NXevent_data";
+	public static final String FERMICHOPPER = "NXfermi_chopper";
+	public static final String FILTER = "NXfilter";
+	public static final String FLIPPER = "NXflipper";
+	public static final String FRESNELZONEPLATE = "NXfresnel_zone_plate";
+
+	/**
+	 * @deprecated superseded by {@link #TRANSFORMATIONS}
+	 */
+	@Deprecated
+	public static final String GEOMETRY = "NXgeometry";
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)